script.js

```diff
@@ -2,19 +2,20 @@ function createCalendar(ID, year, month) {
   let html = '<tr><th>ПН</th><th>ВТ</th><th>СР</th><th>ЧТ</th><th>ПТ</th><th>СБ</th><th>ВС</th></tr>'
   let temp = '';
   let date = new Date(year, month - 1);
-  let firstDay = date.getDay();
 
+  let firstDay = date.getDay() === 0 ? 7 : date.getDay();
```

Comment:
ok here, but usually will be better to save `date.getDay()` in a variable to not do the operation twice. It may be expensive.